### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by dwillson**
Prevented div wrapping in the details div on Edit Profile page and fixed Odo title on Profiles and Request History pages.

Details div now stays fixed next to table when scrolling in Chrome, Firefox, Safari.

Removed destination placeholder and friendly name change text input.

Added helper popup when reordering for paths is toggled on.

Corrected an error that occurs when typing in a destination hostname.

History's page title now shows what profile it is associated with.

Added profile name to historyList on Request History page.

Path group configuration is in a checklist now.

Added Select button to each row of Client list; removed need for text input to change clients.

Temporarily made the details div bigger. Still trying to figure out if
there's a way to make the width fill up remaining space.

If client has no friendly name, URL parameter will default to UUID.

Group selection for a path now fixed; added reset before adding selections.

Fixed bug where clients could not have friendly name being edited as Submit  was being clicked.

### CODE REVIEW

**Comment by dwillson** [comment]
These are @tiffchou's changes with a UI quirk fix

File: proxyui/src/main/webapp/WEB-INF/views/clients_part.jsp [hunk 1]
```diff
@@ -72,42 +67,83 @@ function manageClientPopup() {
...
+        "mtype" : "POST"
+    };
+
+    var checked = isChecked($("#clientlist").getCell(id, 'isActive'));
+    $("#clientlist").jqGrid("editRow", id, true, editParameters);
+    document.getElementById(id+"_isActive").checked = checked;
+}
+
+function isChecked(cellValue) {
+  if(cellValue === true || cellValue === 'Yes') {
```
Comment (djrenfro) [inline proxyui/src/main/webapp/WEB-INF/views/clients_part.jsp]: Indentation is off in this function

File: proxyui/src/main/webapp/WEB-INF/views/clients_part.jsp [hunk 2]
```diff
@@ -72,42 +67,83 @@ function manageClientPopup() {
...
+  }
+  return false;
+}
+
+function changeClientSubmit(id) {
+    var active = $("#" + id + "_isActive").is(":checked");
+
+    saveClientRow(id, active);
+
+    $.removeCookie("UUID", { expires: 10000, path: '/testproxy/' });
```
Comment (djrenfro) [inline proxyui/src/main/webapp/WEB-INF/views/clients_part.jsp]: Do we need to re-add the cookie after this line with the new UUID?

Comment (dwillson) [inline proxyui/src/main/webapp/WEB-INF/views/clients_part.jsp]: It will get set when it reloads the page a few lines down

**Comment by dwillson** [comment]
@djrenfro updated

**Comment by djrenfro** [comment]
:+1:

**Pull request merged**